src/poliastro/twobody/orbit.py

```diff
@@ -610,6 +615,155 @@ def parabolic(
         ss = ClassicalState(attractor, p, 1.0 * u.one, inc, raan, argp, nu)
         return cls(ss, epoch, plane)
 
+    @classmethod
+    @u.quantity_input(alt=u.m, inc=u.rad, argp=u.rad, raan=u.rad, arglat=u.rad)
+    def frozen(
+        cls,
+        attractor,
+        alt,
+        inc=None,
+        argp=None,
+        raan=0 * u.deg,
+        arglat=0 * u.deg,
+        epoch=J2000,
+        plane=Planes.EARTH_EQUATOR,
+    ):
+        r"""Return frozen Orbit.
+
+        To achieve frozen orbit these two equations have to be set to zero.
+
+        .. math::
+            \begin{align}
+                \dfrac {d\overline {e}}{dt}=\dfrac {-3\overline {n}J_{3}R^{3}_{E}\sin \left( \overline {i}\right) }{2a^{3}\left( 1-\overline {e}^{2}\right) ^{2}}\left( 1-\dfrac {5}{4}\sin ^{2}\overline {i}\right) \cos \overline {w}\\
+                \dfrac {d\overline {\omega }}{dt}=\dfrac {3\overline {n}J_{2}R^{2}_{E}}{a^{2}\left( 1-\overline {e}^{2}\right) ^{2}}\left( 1-\dfrac {5}{4}\sin ^{2}\overline {i}\right) \left[ 1+\dfrac {J_{3}R_{E}}{2J_{2}\overline {a}\left( 1-\overline {e}^{2}\right) }\left( \dfrac {\sin ^{2}\overline {i}-\overline {e}\cos ^{2}\overline {i}}{\sin \overline {i}}\right) \dfrac {\sin \overline {w}}{\overline {e}}\right]
+            \end{align}
+
+        The first approach would be to nullify following term to zero:
+
+        .. math::
+            ( 1-\dfrac {5}{4}\sin ^{2})
+
+        For which one obtains the so-called critical inclinations: i = 63.4349 or 116.5651 degrees. To escape the inclination requirement, the argument of periapsis can be set to w = 90 or 270 degrees to nullify the second equation. Then, one should nullify the right-hand side of the first equation, which yields an expression that correlates the inclination of the object and the eccentricity of the orbit:
+
+        .. math::
+            \overline {e}=-\dfrac {J_{3}R_{E}}{2J_{2}\overline {a}\left( 1-\overline {e}^{2}\right) }\left( \dfrac {\sin ^{2}\overline {i}-\overline {e}\cos ^{2} \overline {i}}{\sin \overline {i}}\right)
+
+        Assuming that e is negligible compared to J2, it can be shown that:
+
+        .. math::
+            \overline {e}\approx -\dfrac {J_{3}R_{E}}{2J_{2}\overline {a}}\sin \overline {i}
+
+        The implementation is divided in the following cases:
+
+        1- When the user gives a negative altitude, the method will raise a ValueError
```

Comment:
This list was not rendered correctly though: https://2744-11178845-gh.circle-artifacts.com/0/docs-html/api/safe/twobody/orbit.html#poliastro.twobody.orbit.Orbit.frozen